PrivateCloud.DiagnosticInfo/PrivateCloud.DiagnosticInfo.psm1

```diff
@@ -710,6 +710,40 @@ function Start-CopyJob(
     }
 }
 
+#
+# Utility wrapper for invoking commands by opening sessions 
+# for each of the cluster nodes and preserving the session 
+# to be deleted after use.
+#
+
+function New-InvokeCommand (

```

Comment:
We should keep using the Invoke verb, some different noun like SddcCommonCommand. "New" isn't right in this context, thats creation/instantiation, not alternate/different.

Reply:
Invoke-SddcCommonCommand sounds good?

Reply:
Predict my answer, but I was intending that as a suggestion, not strictly a prescription ... sure, though.